anonymizer/base.py

```diff
@@ -247,16 +249,33 @@ class Anonymizer(object):
 
     faker = DjangoFaker()
 
+    def __init__(self):
+        super(Anonymizer, self).__init__()
+
+        assert self.attributes is not None, '"attributes" attribute must be set'
```

Comment:
These attributes should already be set during class construction time, so now they're just done once instead of to every call to `get_queryset` or the like